src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/GeographicRegion.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.model;
+
+import jakarta.persistence.DiscriminatorValue;
+import jakarta.persistence.Entity;
+import lombok.NoArgsConstructor;
+
+@Entity
+@DiscriminatorValue(GeographicRegion.GEOGRAPHIC)
+@NoArgsConstructor
+public class GeographicRegion extends ReferenceCode {
```

Comment:
new java class 😱

Reply:
well ideally any new files should be kotlin unless there is a reason for them to be java

Reply:
since we are gradually converting all to kotlin anyway

Reply:
I'll do it in the next commit